about.html

```diff
@@ -40,6 +67,64 @@ <h1>DERMATOLOGY CLINIC & SKINCARE CONSULTATION CENTER</h1>
             <a href="favourezeugwa@gmail.com">favourezeugwa@gmail.com</a>
         </div>
     </section>
+
+    <!-- Company Logo decision section -->
+    <section id="main-logo">
+        <h3>OUR MEDICAL <br>
+            CONSULTATION CENTER LOGO</h3>
+        <hr>
+        <p>The logo of our medical consultation center was decided through the logo competition from 8th April to 7th
+            June.</p>
+        <div class="company-logo">
+            <img src="./images/flower-logo1.jpg" alt="company logo">
+            <h2><b>DS Consultation and Medical center 2022.</b></h2>
+        </div>
+    </section>
+
+    <!-- client testimonials -->
+    <section id="client-reviews">
+        <h3>Reviews from our satisfied clients</h3>
+        <hr>
+        <p>Take a look at the before and after photographs of some of our amazing clients.</p>
+        <div class="client-container">
+            <div class="client-img">
+                <img src="./images/client-7.jpg" alt="client image">
+                <div class="client-card">
+                    <h4><b>Kingston Ike. 18th,Jan</b></h4>
+                    <p><i><b>'It's amazing how my skin has rejuvenated just 1 week after meeting with Dr
+                                clinton'</b></i>
+                    </p>
+                </div>
+            </div>
+            <div class="client-img">
+                <img src="./images/client-review3.webp" alt="client image">
+                <div class="client-card">
+                    <h4><b>Claire Amos. 20th,April</b></h4>
+                    <p><i><b>'I just have a very positive experience with your clinic, Thank you Favour!</b></i></p>
+                </div>
```

Comment:
#### Well done with this project 💯 

- [ ] Just your images are overflowing on the tablet screens please try to fix that in order to keep your website responsive on all devices
![image](https://user-images.githubusercontent.com/61790587/169650276-e665955f-22ba-4d5f-92d4-e3fd47af7bfa.png)